client/my-sites/upgrades/domain-management/dns/dns-add-new.jsx

```diff
@@ -95,7 +98,7 @@ const DnsAddNew = React.createClass( {
 				if ( error ) {
 					notices.error( error.message );
```

Comment:
I only reduxified "Simple" cases. 
Global redux notices don't support embedded components yet and we try to avoid it.
In this particular case, notice comes from `addDns` callback which I havent figured out yet, and I will be trying that out once this PR is merged :)
this PR is already a month old and got derailed with some philosphical redux conversation :)

Thanks! :)